packages/annotations/package.json

```diff
@@ -9,12 +9,15 @@
   "types": "dist/esm/index.d.ts",
   "license": "Apache-2.0",
   "dependencies": {
-    "@esri/arcgis-rest-auth": "^1.11.0",
-    "@esri/arcgis-rest-common-types": "^1.11.0",
-    "@esri/arcgis-rest-feature-service": "^1.11.0",
-    "@esri/arcgis-rest-items": "^1.11.0",
-    "@esri/arcgis-rest-request": "^1.11.0",
-    "@esri/arcgis-rest-users": "^1.11.0",
+    "@esri/arcgis-rest-auth": "^1.13.1",
```

Comment:
so, below other rest-js packages are updated to `1.12.0`, should we bump them to `1.13.1`?